ciao_contrib/_tools/merging.py

```diff
@@ -507,6 +507,21 @@ def update_ancillary_keywords(infile, outfile, asolfile):
 #            # do this.
 #            # update_ancillary_keywords(infile, outfile, asolfile)
 
+def _copy_region_block(infile, outfile):
+    """DM bug with dmcopy opt=all means we need to avoid it and
+    copy any REGION blocks manually (gratings data)"""
+    from ciao_contrib.runtool import dmappend
+
+    inblk = pycrates.read_file(infile)
+    in_ds = inblk.get_dataset()
+    try:
+        regblk = in_ds.get_crate("REGION")
+    except IndexError as no_region:
```

Comment:
Since we are ignoring the exception we can just say

```suggestion
    except IndexError:
```